lib/ecl/ecl_smspec.cpp

```diff
@@ -1287,18 +1287,28 @@ int ecl_smspec_get_num_regions(const ecl_smspec_type * ecl_smspec) {
 
 /*****************************************************************/
 
-#define NODE_RETURN_INDEX(node_ptr)                     \
-  if (!node_ptr)                                        \
-    return -1;                                          \
-  else                                                \
-    return smspec_node_get_params_index( node_ptr );
 
 
-#define NODE_RETURN_EXISTS(node_ptr)            \
-  if (!node_ptr)                                \
-    return false;                               \
-  else                                          \
-    return true;
+
+namespace {
+
+  bool NODE_EXISTS(const ecl::smspec_node * node_ptr) {
```

Comment:
Should we make these styled as function names instead of MACROS?

Reply:
Yes, that makes sense - I just wanted to keep the amount of change down. Will update the PR